dashboard/src/main/resources/css/dashboard.css

```
@@ -43,20 +43,84 @@
  }
  
  h1 {
-   font-size: 36pt;
+   font-size: 3em;
  }
  
  h2 {
-   font-size: 30pt;
+   font-size: 2.5em;
  }
  
  h3 {
-   font-size: 24pt;
+   font-size: 2em;
  }
+ 
  th, td {
    padding: 5pt;
  }
  
  pre {
    line-height: normal;
- }
\ No newline at end of file
+ }
+ 
+
+ /* ----- Statistic ----- */
+.statistics {
+    padding-top: 50px;
+}
+
+.container {
+  min-height: 20ex;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+}
+
+.statistic_item {
```

```suggestion
.statistic-item {
```

Here an underscore is used to separate words, but for all these CSS selectors for statistics, shall we start following this rule?

> Separate words in ID and class names by a hyphen.
https://google.github.io/styleguide/htmlcssguide.html#CSS_Style_Rules